### ISSUE TITLE
Items With No Apparent Use

### ISSUE THREAD

**Issue by ColdStoneSys** [TonimatasMCDEV/KrystalCraft#16]
There are a few items like the gemstone dusts (lapis, ruby, sapphire, etc) that don't seem to have any apparent use. 

Respectfully, What is the point of their inclusion? 

Maybe I'm missing something.

**Comment by TonimatasDEV** [comment]
Yes, they are not currently useful (only for armor and tools), but KrystalCraft is not finished. However, version 1.21 stopped development because KrystalCraft libraries are not being updated ([Botarium](https://www.curseforge.com/minecraft/mc-mods/botarium) and [Resourceful Lib](https://www.curseforge.com/minecraft/mc-mods/resourceful-lib/files))

**Comment by ColdStoneSys** [comment]
I see, thank you for the explaination.
